specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -1,8 +1,8 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "ArtifactsClient",
-    "description": "SyMS API Service Client.",
+    "title": "SyMS API Client",
```

Comment:
Please keep "title" same as before otherwise it may bring breaking for SDK.

Reply:
Okay. Reverted.